diffsmhm/tests/testing_catalogs/make_mock_halos.py

```diff
@@ -0,0 +1,195 @@
+import numpy as np
+import h5py
+
+import os
+
+
+def _write_file(fname, hid, upid, pid,
+                mpeak, host_mpeak,
+                x, y, z, hx, hy, hz,
+                host_dist):
+
+    with h5py.File(fname, "w") as f:
+        f.create_dataset("halo_id", data=hid)
+        f.create_dataset("upid", data=upid)
+        f.create_dataset("pid", data=pid)
+
+        f.create_dataset("mpeak", data=mpeak)
+        f.create_dataset("host_mpeak", data=host_mpeak)
+
+        f.create_dataset("x", data=x)
+        f.create_dataset("y", data=y)
+        f.create_dataset("z", data=z)
+
+        f.create_dataset("host_x", data=hx)
+        f.create_dataset("host_y", data=hy)
+        f.create_dataset("host_z", data=hz)
+
+        f.create_dataset("host_dist", data=host_dist)
+
+
+def make_test_catalogs(t):
+    # define file names
+    script_dir = os.path.realpath(os.path.dirname(__file__))
+    filenames = [
+                    script_dir+"/mock_halos_1.h5",
+                    script_dir+"/mock_halos_2.h5",
+                    script_dir+"/mock_halos_3.h5",
+                    script_dir+"/mock_halos_4.h5",
+                    script_dir+"/mock_halos_5.h5"
+                ]
+
+    # remove any existing files
+    for fname in filenames:
+        if os.path.exists(fname):
+            os.remove(fname)
+
+    if t == 1:
```

Comment:
Instead of numbers here, we should use descriptive strings on what each catalog is designed to test.

Reply:
A few more comments would be helpful too.